packages/split-view/test/split-view.test.ts

```diff
@@ -0,0 +1,1035 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { fixture, elementUpdated, expect, html } from '@open-wc/testing';
+
+import '../sp-split-view.js';
+import { SplitView } from '../';
+import {
+    arrowDownEvent,
+    arrowLeftEvent,
+    arrowRightEvent,
+    arrowUpEvent,
+    endEvent,
+    homeEvent,
+    pageDownEvent,
+    pageUpEvent,
+    shiftTabEvent,
+} from '../../../test/testing-helpers.js';
+import { spy } from 'sinon';
+
+describe('SplitView', () => {
+    it('loads default (horizontal) split-view accessibly', async () => {
+        const el = await fixture<SplitView>(
+            html`
+                <sp-split-view primary-size="100">
+                    <div>First panel</div>
+                    <div>Second panel</div>
+                </sp-split-view>
+            `
+        );
+
+        await elementUpdated(el);
+        await expect(el).to.be.accessible();
+        expect(el.splitterPos || 0).to.equal(100);
+        expect(el.resizable).to.be.false;
+        expect(el.collapsible).to.be.false;
+        const gripper = el.shadowRoot.querySelector(
+            '#gripper'
+        ) as HTMLDivElement;
+        // expect(gripper).not.to.be.accessible();
+        expect(gripper).to.be.null;
+        const splitter = el.shadowRoot.querySelector(
+            '#splitter'
+        ) as HTMLDivElement;
+        expect(getComputedStyle(splitter).cursor).to.equal('auto');
+    });
+
+    it('loads horizontal [resizable] split-view accessibly', async () => {
+        const el = await fixture<SplitView>(
+            html`
+                <sp-split-view resizable primary-size="100px">
+                    <div>First panel</div>
+                    <div>Second panel</div>
+                </sp-split-view>
+            `
+        );
+
+        await elementUpdated(el);
+        await expect(el).to.be.accessible();
+        expect(el.splitterPos || 0).to.equal(100);
+
+        const gripper = el.shadowRoot.querySelector(
+            '#gripper'
+        ) as HTMLDivElement;
+        expect(gripper).to.be.accessible();
```

Comment:
`await` or if the linter complains move to something like `.to.not.be.null`

Reply:
awaited all